pages.th/osx/pbcopy.md

```diff
@@ -0,0 +1,12 @@
+# pbcopy
+
+> คัดลอกข้อมูลจากอินพุตมาตรฐาน (`stdin`) ไปยังคลิปบอร์ด ซึ่งเทียบได้กับการกดปุ่ม Cmd + C บนแป้นพิมพ์.
+> ข้อมูลเพิ่มเติม: <https://ss64.com/osx/pbcopy.html>.
+
+- คัดลอกเนื้อหาในไฟล์ที่กำหนดไปยังคลิปบอร์ด:
+
+`pbcopy < {{ที่อยู่ของไฟล์}}`
```

Comment:
```suggestion
`pbcopy < {{ทาง/ไป/ไฟล์}}`
```
This is what we use for this placeholder across all the pages. [Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)